third_party/move/tools/move-formatter/tests/formatter/tuple/input5.move.fmt

```diff
@@ -0,0 +1,34 @@
+address 0x42 {
+    module example {
+        /**
+        * This function doesn't return anything. It's just for demonstration purposes. 
```

Comment:
Not sure why we introduced a line break inside of the comment: we should not?

Reply:
Updated. According to the idea of the above comment, we will not modify the `comment`